pkg/kubelet/util/csr/csr.go

```diff
@@ -39,7 +49,7 @@ import (
 func RequestNodeCertificate(client certificatesclient.CertificateSigningRequestInterface, privateKeyData []byte, nodeName types.NodeName) (certData []byte, err error) {
```

Comment:
ok, and since every rotation attempt uses a new key, reentrancy isn't as important (though the switch to WatchUntil would be good for it to pick up in 1.8)